projects/aca-content/src/lib/components/viewer/viewer.module.ts

```diff
@@ -34,21 +33,10 @@ import { CoreExtensionsModule } from '../../extensions/core.extensions.module';
 import { AppToolbarModule } from '../toolbar/toolbar.module';
 import { AppViewerComponent } from './viewer.component';
 
-const routes: Routes = [
-  {
-    path: '',
-    data: {
-      title: 'APP.PREVIEW.TITLE',
-      navigateMultiple: true
-    },
-    component: AppViewerComponent
-  }
-];
-
 @NgModule({
   imports: [
     CommonModule,
-    RouterModule.forChild(routes),
+    // RouterModule.forChild(routes),
```

Comment:
commented code again

Reply:
@DenysVuika because this is a draft PR :) I am using draft to see all changes and before remove the draft state, I am verifying everything what I did. I would not expect everything clean in draft PR